hwdb.d/70-mouse.hwdb

```diff
@@ -143,6 +143,34 @@ mouse:*:name:*trackball*:*
 mouse:*:name:*TrackBall*:*
  ID_INPUT_TRACKBALL=1
 
+##########################################
+# 3D Connexion
+##########################################
+# From https://github.com/FreeSpacenav/spacenavd/blob/39856625a6de1e8c4b57c5938e1bf29d13cf1a9f/src/dev.c#L63
+mouse:usb:v046dpc603:name:*:*
```

Comment:
fwiw I think skipping the `name:*:*` in favour of just `*` would do the job, seems superfluous to specify it (even though some others here like the apple magic mouse do it too).

Reply:
Copy/paste cargo culting. Fixed.